buildSrc/src/main/resources/windowsStartScript.txt

```diff
@@ -0,0 +1,93 @@
+@rem
```

Comment:
Do we still want to check in this file?

Reply:
I could go either way - if we leave it in, maybe i add another comment in the edited region to note that this isn't usable yet, though I think the syntax at least is right.

Removing the file would mean we'd need to re-implement it - I could copy the code and put it somewhere else, but "where it belongs" seems better?

Will go either way, based on your opinion.

Reply:
Let's take it out for now - I think it's easy to re-impl when necessary.